### PR TITLE
SN-API-9 Add logger

### PR DESCRIPTION
**Pull request by Aretemal**

### CODE REVIEW

File: src/logger/logger-history/sequelize.log [hunk 1]
```diff
@@ -0,0 +1,5 @@
+time: 2023-04-19T11:17:44.161Z 
+info: 
+Executing (default): SELECT `id`, `firstName`, `lastName`, `password`, `email`, `login`, `
+status`, `ava`, `createdAt`, `updatedAt` FROM `Users` AS `User` WHERE `User`.`login` = 'us
+er1' LIMIT 1;
```
Comment (Konstantsin) [inline src/logger/logger-history/sequelize.log:5]: Do u really need logs in your repo?

File: src/controllers/PostController.js [hunk 2]
```diff
@@ -23,7 +25,8 @@ class PostController {
   async getOne(req, res, next) {
     const errors = validationResult(req);
     if (!errors.isEmpty()) {
-      return res.status(400).json({ errors: errors.array() });
+      const errorString = errors.array().map((item) => item.msg).join(', ');
+      throw new AppError(errorString, 400);
```
Comment (Konstantsin) [inline src/controllers/PostController.js:29]: Is it a good practice to pass validation errors to the error handler by raising AppError. 
A validation error is not really an error. At the level of the user, it's an error, but at the application level, this is a manageable situation.  Having `ErrorSerializer` will help. 
Please check the standard for error response and implement it.

File: src/middlewaree/responseJSON.js [hunk 3]
```diff
@@ -1,4 +1,10 @@
+import logger from '../logger/logger.js';
+
 export const responseJSON = (req, res, next) => {
+  logger.successLogger.log(
+    'info',
+    `Request ${req.protocol}://${req.get('host')}${req.originalUrl} completed successfully`,
+  );
```
Comment (Konstantsin) [inline src/middlewaree/responseJSON.js:7]: What if serialization in the next line crashes? 
Maybe we should log success when the render is already done? 
Maybe even in a different middleware?

File: src/logger/logger-history/sequelize.log (hunk 1 above)
Comment (Aretemal) [reply src/logger/logger-history/sequelize.log:5]: done

File: src/middlewaree/responseJSON.js (hunk 3 above)
Comment (Aretemal) [reply src/middlewaree/responseJSON.js:7]: done

**Pull request merged**